server.py

```diff
@@ -663,6 +670,25 @@ def post(self, region, id):
         except:
             return 'Dao threw an error somewhere', 400
 
+class ExcludeTournamentMatchResource(restful.Resource):
+    def get(self, region, id):
+        pass
+
+    def post(self, region, id):
+        dao = Dao(region, mongo_client=mongo_client)
+        user = get_user_from_request(request, dao)
+        #if not user:
+        #    return 'Permission denied', 403
+        #if not is_user_admin_for_regions(user, region):
+        #    return 'Permission denied', 403
+
+        args = tournament_details_exclude_match_parser.parse_args()
+        tournament_id = args['tournament_id']
+        match_id = args['match_id']
```

Comment:
since you cast tournament_id to an ObjectId and excluded to a bool here, i would also cast match_id to an int here instead of in dao.set_match_exclusion_by_etc.